### PR TITLE
PT-3939: Include whitespace separator in parsing multiple emails in ContactInfo

### PR DESCRIPTION
**Pull request by veronikaslc**
Added a blank space to possible separators

### CODE REVIEW

File: components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java [hunk 1]
```diff
@@ -225,7 +225,7 @@ private void setEmails(List<String> emails)
             {
                 List<String> collectedEmails = new ArrayList<String>();
                 for (String email : emails) {
-                    for (String parsedEmail : StringUtils.split(email, ",|;")) {
+                    for (String parsedEmail : StringUtils.split(email, ",|;| ")) {
```
Comment (marta-) [inline components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java]: Afaik the second parameter of `split` which specifies the separators is treated as a list of characters, not a regular expression. You should just add the space, there's no need for another `|`.

**Comment by allasm** [comment]
PT-3882 is closed and included in some already released versions. I think this should be under  anew PT issue, not PT-3882

File: components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java (hunk 1 above)
Comment (allasm) [reply components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java]: I think Marta is right (https://github.com/phenotips/phenotips/pull/2815#issuecomment-460821455), and I would remove `|` altogether as a highly unlikely separator

Comment (veronikaslc) [reply components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java]: `|` serves as an `OR` in this case

Comment (allasm) [reply components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java]: But it looks like it does not, from the specification of this function

Comment (veronikaslc) [reply components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java]: will remove it then

**Comment by veronikaslc** [comment]
Closed in favour of https://github.com/phenotips/phenotips/pull/2816

**Pull request closed without merging**